lib/src/icons.dart

```diff
@@ -11,376 +11,386 @@ class GYWIcon {
   /// The height (in pixels) of the icon
   final double height;
 
+  /// The path to the PNG file
+  final String pathPng;
+
+  /// The path to the SVG file
+  final String pathSvg;
+
   const GYWIcon({
     required this.name,
     required this.filename,
     required this.width,
     required this.height,
+    this.pathPng = "assets/icons/unknown.png",
+    this.pathSvg = "assets/icons/unknown.png",
```

Comment:
Is this icon already in the assets folder ?

Reply:
Whoops. I thought the icon already existed.
Should we have a dedicated icon for unknown icons, or should we just set it to `help.png`.